### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by maddenlockin**

### CODE REVIEW

**Review by maddenlockin — COMMENTED** [review]
reviewed

File: app.js [hunk 1]
```diff
@@ -1,8 +1,54 @@
...
 // let state
+let wins = 0;
+let total = 0;
+
+disImage0.src = 'assets/cup.png';
+disImage1.src = 'assets/cup.png';
+disImage2.src = 'assets/cup.png';
+
+// create handler function for user guess
+function handleGuess(id) {
```
Comment (Bradley-Bird) [inline app.js:20]: We want the handleGuess funciton to to take 2 parameters-- handleGuess(userGuess, correctSpot)

File: app.js [hunk 2]
```diff
@@ -1,8 +1,54 @@
...
+    // set display image src
+    disImage0.src = 'assets/cup.png';
+    disImage1.src = 'assets/cup.png';
+    disImage2.src = 'assets/cup.png';
 
+    // increment total
+    total++;
+    // randomize app cup choice & compare
+    const randomizeBall = Math.floor(Math.random() * 3);
```
Comment (Bradley-Bird) [inline app.js:29]: this logic should be done outside of the handler function.

**Review by Bradley-Bird — COMMENTED** [review]
Take a look at the functions sections of the rubric and see if you can refactor this to meet the requirements.

File: app.js [hunk 3]
```diff
@@ -1,8 +1,54 @@
...
+const disImage2 = document.getElementById('cup-2');
+const winsEl = document.getElementById('wins');
+const lossesEl = document.getElementById('losses');
+const totalEl = document.getElementById('total');
+const button0 = document.getElementById('button-0');
+const button1 = document.getElementById('button-1');
+const button2 = document.getElementById('button-2');
 // let state
+let wins = 0;
+let total = 0;
```
Comment (maddenlockin) [inline app.js:13]: thats the total

File: index.html [hunk 4]
```diff
@@ -13,60 +13,23 @@
...
-            </p>
-
+        <section class="images">
+            <img id="cup-0" src="" />
+            <img id="cup-1" src=""/>
+            <img id="cup-2" src=""/>
+        </section>
+        <section class="buttons">
+            <button id="button-0">This Cup!</button>
+            <button id="button-1">This Cup!</button>
```
Comment (maddenlockin) [inline index.html:24]: comment

**Review by maddenlockin — COMMENTED** [review]
all done

File: index.html [hunk 5]
```diff
@@ -13,60 +13,23 @@
...
+            <button id="button-2">This Cup!</button>
+        </section>
+        <section class="stats">
+            <div class="stat" id="wins">Wins:</div>
+            <div class="stat" id="losses">Losses:</div>
+            <div class="stat" id="total">Total:</div>
```
Comment (belliott15) [inline index.html:30]: Great job with your naming and sharing a class.  Try and remove any empty space to make it a little better readability.

**Review by belliott15 — APPROVED** [review]

File: app.js [hunk 6]
```diff
@@ -1,8 +1,54 @@
...
+button0.addEventListener('click', () => {
+    handleGuess(0);
+});
+button1.addEventListener('click', () => {
+    handleGuess(1);
+});
+button2.addEventListener('click', () => {
+    handleGuess(2);
+});
+
```
Comment (delaneyfogarty) [inline app.js:54]: Looks like you are missing the reset functionality!

**Review by delaneyfogarty — APPROVED** [review]

**Pull request merged**